code_map.py

```diff
@@ -601,10 +629,16 @@ def run(self, edit, direction=None, start=False, stop=False, fast=False):
 
 
 class synch_code_map(sublime_plugin.TextCommand):
-
+    '''This command also activates the CodeMap if the view is present, but
+    CodeMap is inactive because ST has just been restarted, or the project has
+    been switched'''
     # -----------------
 
     def run(self, edit, from_view=False):
+        global ACTIVE
+
+        if not ACTIVE and get_code_map_view():
+            ACTIVE = True
 
```

Comment:
I modified the synch map command, so that it activates the map if the panel is open but the map is inactive. Since it has become more important, I'd prefer it mapped at `alt+m, alt+m`, at least I map it there.